finalized-tag-updater/src/main/kotlin/net/consensys/linea/FinalizationUpdatePoller.kt

```diff
@@ -49,7 +50,12 @@ class FinalizationUpdatePoller(
   }
 
   override fun action(): SafeFuture<*> {
-    return lineaRollup.currentL2BlockNumber().sendAsync()
+    return AsyncRetryer.retry(
+      vertx,
+      backoffDelay = config.pollingInterval
+    ) {
+      SafeFuture.of(lineaRollup.currentL2BlockNumber().sendAsync())
```

Comment:
Add retry here to wait until currentL2BlockNumber on L1 is present to avoid getting the "Empty value (0x) returned from contract" warning message.